packages/create-novu-app/templates/index.ts

```diff
@@ -175,7 +197,10 @@ export const installTemplate = async ({
     version: "0.1.0",
     private: true,
     scripts: {
-      dev: "next dev --port=4000",
+      tunnel:
+        "node --loader ts-node/esm --no-warnings=ExperimentalWarning scripts/tunnel.mts",
+      "next-dev": `next dev --port=${port}`,
+      dev: "concurrently 'npm run tunnel' 'npm run next-dev'",
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
⚠ **issue:** It feels a bit heavy-handed and opinionated to add a library to concurrently run the tunnel with Dev server. Are there `next dev` scripts we can hook into to setup the tunnel in a more transparent way?‏

Reply:
discussed 1on1, decided to go with `concurrently`